app/models/user.rb

```diff
@@ -84,8 +84,13 @@ class User < ApplicationRecord
   scope :not_signed_in, -> { where(last_sign_in_at: nil, active: true) }
   scope :deactivated, -> { where(active: false) }
   scope :active_status, -> { where(active: true).where.not(last_sign_in_at: nil) }
-  scope :visible, -> { where(discarded_at: nil) }
-  scope :own_and_managing_org_users, ->(organisation) { where(organisation: organisation.child_organisations + [organisation]) }
+  scope :visible, lambda { |user = nil|
+    if user && !user.support?
+      where(discarded_at: nil, organisation: user.organisation.child_organisations + [user.organisation])
+    else
+      where(discarded_at: nil)
+    end
+  }
```

Comment:
I've made passing a user parameter optional for the visible scope. Should a user be passed in and they are not a support user their scope will be filtered by organisation otherwise if they're a support user or the visible scope is used without a user passed all non-discarded records will be returned.